webpack/envs/serverConfig.js

```diff
@@ -62,18 +62,16 @@ export const serverConfig = {
     minimize: env.isProduction && !env.webpackDebug,
     minimizer: [
       new TerserPlugin({
-        cache: false,
```

Comment:
(nonblocking) This significantly speeds up local prod builds, can we perhaps disable it if we're in a deploy environment?

Reply:
It's literally no longer an option and has been integrated with their new caching system.